### ISSUE TITLE
CVC not cleared on stored payment method when an error occurs

### ISSUE THREAD

**Issue by Manny-ModMed** [Adyen/adyen-ios#403]
**Describe the challenge**
A clear and concise description of what you are trying to achieve.
What steps did you take?
What is the expected behavior / API ?

Hello, I am currently addressing an issue regarding stored payment methods. When a user selects a stored payment method in the drop down component, an alert is shown that will ask the user to enter their CVC (I believe this is being managed by `StoredCardAlertManager`). When the user enters the wrong CVC or a payment error occurs on the server, we show an alert with the error that occurred to the user. Now if the user decides the select the same stored payment method that failed, the alert to ask the user to enter their CVC appears, but the CVC field is already populated with the previous entered CVC. So far, I haven't found an API that allows us to clear the CVC field in the alert. Is there a way to clear the CVC field when an error occurs? 

**Documentation update (optional)**
N/A

**Screenshots**
I've attached a video of the current behavior that we are experiencing :
https://user-images.githubusercontent.com/41126742/111008743-c4187100-835f-11eb-9221-2b62666e6621.mp4

**Comment by mohammedDehairy** [comment]
Hi @Manny-ModMed ,

Thanks for reporting this!.

 We can simply clear sensitive fields when a component is dismissed like `CVC` and `PAN`, but I think it makes sense to keep fields like email address and names ..etc, do you agree?.

**Comment by Manny-ModMed** [comment]
@mohammedDehairy 

I agree with clearing sensitive fields like the `CVC` and `PAN` fields. But I'm not sure about fields like email address, names, etc. Maybe for those, we could have a configuration to opt in to clearing out.

**Comment by mohammedDehairy** [comment]
@Manny-ModMed ,

`Maybe for those, we could have a configuration to opt in to clearing out.`

Yeah, that's also a good idea.

**Comment by Manny-ModMed** [comment]
@mohammedDehairy 

For the `CVC/PAN` fields, is this something that can be implemented?

**Comment by mohammedDehairy** [comment]
Hi @Manny-ModMed ,

Yes of course, no worries, its on our radar, and will be done in the next release(s).

**Comment by descorp** [comment]
Fixed in [4.6.0](https://github.com/Adyen/adyen-ios/releases/tag/4.6.0)

@Manny-ModMed could you verify?

**Comment by descorp** [comment]
I am closing this issue due to lack of activity
